packages/angular/src/generators/init/init.ts

```diff
@@ -53,13 +54,17 @@ export async function angularInitGenerator(
     : () => {};
   const unitTestTask = await addUnitTestRunner(tree, options);
   const e2eTask = addE2ETestRunner(tree, options);
+  const jsInitTask = jsInitGenerator(tree, {
```

Comment:
Everywhere we call this from another generator, we probably pass `skipFormat`. Otherwise, formatting will happen multiple times and be slow.

Reply:
ok. added skipFormat to jsInitGenerator.